addons/viewport/src/Tool.tsx

```diff
@@ -196,10 +196,10 @@ export const ViewportTool: FunctionComponent = memo(
             <Global
               styles={{
                 [`#${iframeId}`]: {
-                  margin: `auto`,
+                  margin: `${theme.layoutMargin}px auto`,
                   transition: 'width .3s, height .3s',
                   position: 'relative',
-                  border: `${theme.layoutMargin}px solid black`,
+                  border: `1px solid black`,
                   borderRadius: theme.appBorderRadius,
```

Comment:
A single px border + 10px borderRadius together will mean we'll crop parts of the user's iframe.. I didn't want to do that, hence the border.

The user will not be able to click on the corners of their component, when using this addon.

Reply:
Fair point that I hadn't considered 👍 What if we simply drop the border-radius?

Reply:
That'd be a design decision I'd like @domyen's opinion on